src/components/Popup/Popup.scss

```diff
@@ -0,0 +1,82 @@
+@use "../../styles/variables" as *;
+
+.popup {
+  // TODO: Remove bg color styles
+
+  background-color: rgba($color: #000000, $alpha: 0.5);
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__container {
+    width: 100%;
```

Comment:
I think we will want to put a max width to limit it from spanning the whole of the screen